workspaces/shorturl/backstage.json

```diff
@@ -0,0 +1 @@
+{ "version": "1.30.0" }
```

Comment:
This tells me you are still on an older version which is probably why the API Reports are not coming up correctly

Reply:
You should be able to run `yarn backstage-cli versions:bump --release main` from the root of your workspace: '`/workspaces/shorturl`. Then commit those changes. Once that's done then try building the API Reports again with  `yarn build:api-reports`

Reply:
got it. I was running it in the root. Now it worked, thanks. Can you re-trigger build for me?